xdsl/dialects/scf.py

```diff
@@ -689,6 +697,107 @@ def parse(cls, parser: Parser) -> Self:
         return op
 
 
+@irdl_op_definition
+class IndexSwitchOp(IRDLOperation):
+    name = "scf.index_switch"
+
+    arg = operand_def(IndexType)
+    cases = attr_def(DenseArrayBase)
+
+    output = var_result_def(AnyAttr())
```

Comment:
```suggestion
    output = var_result_def()
```

Reply:
Also removed all the other AnyAttr's in this file